src/queries/__tests__/displayValue.test.tsx

```diff
@@ -119,3 +119,29 @@ test('byDisplayValue queries support hidden option', () => {
     `"Unable to find an element with displayValue: hidden"`
   );
 });
+
+test('byDisplayValue should return composite component if useLegacyQueries option is true', () => {
```

Comment:
Agree that we should test both available behaviours, but probably should treat the legacy one as default one, and the duplicate the existing tests and make necessary changes when the are non-legacy. It might make sense just to duplicate the whole file and add some suffix to it: `displayValue.test.tsx` => `displayValue.test.tsx ` + `displayValue.breaking.test.tsx`. Wdyt? It would be easier to clean up after major release when the breaking change will become the new default.

Reply:
Yes I like the idea, i'll change it